src/signals/models.py

```diff
@@ -126,6 +126,32 @@ def __str__(self) -> str:
         return self.name
 
 
+class SignalGroup(models.Model):
+    """
+    A model representing a signal group.
+    Is a representative group for a small group of signals
+    """
+    name = models.CharField(
+        help_text=_('Name'),
+        max_length=128,
+        unique=True
+    )
```

Comment:
if you go with the recommendation below, we'd also need an fk here

```suggestion
    name = models.CharField(
        help_text=_('Name'),
        max_length=128,
        unique=True
    )
    base = models.ForeignKey(
        'signals.Signal',
        help_text=_('Base Signal'),
        related_name='base_for',
        on_delete=models.PROTECT,
        null=True
    )
```

Reply:
didn't get from what column in the g.sheet SignalGroup.base should be getted?

Reply:
The pair (Source Subdivision, Signal BaseName) in the Signals g.sheet uniquely identifies the base signal for each signal group. All rows of the Signals g.sheet that have the same pair are members of the same group.